docs/index.md

```diff
@@ -31,64 +31,130 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
 
-[Learn more](./usecases/data-devenv.md){:id="user-content-learn-more-env"}
 
-### During Deployment
-* **Version Control** - retain commits for a configurable duration, so readers are able to query data from the latest commit or any other point in time. Writers atomically introduce new data preventing inconsistent data views.
-* **Test** - define pre-merge and pre-commit hooks to run tests that enforce schema and validate properties of the data to catch issues before they reach production.
+## Why is lakeFS the data solution you've been missing?
 
-[Learn more](./usecases/ci.md){:id="user-content-learn-more-int"}
+Working with data in a lakeFS repository––as opposed to a bucket––enables simplified workflows when developing data lake pipelines.
 
-### In Production
-* **Roll Back** - recover from errors by instantly reverting data to a former, consistent snapshot of the data lake. Choose any commit in a repository's commit history to revert in one atomic action.
-* **Troubleshoot** - investigate production errors by starting with a snapshot of the inputs to the failed process. Spend less time re-creating the state of datasets at the time of failure, and more time finding the solution.
-* **Cross-collection Consistency** - provide consumers multiple synchronized collections of data in one atomic, revertable action. Using branches, writers provide consistency guarantees across different logical collections - merging to the main branch only after all relevant datasets have been created or updated successfully.
-   
-## Additional things you should know about lakeFS: 
-* It is **format agnostic**
-* Your data **stays in place**
-* It helps you prevent data duplication by using **copy-on-write**
-* It is **highly performant** for huge data lakes
-* It includes **configurable garbage collection** capabilities
-* lakeFS is **highly available and production ready**
+Never again will you spend time doing the following tasks:
 
-<img src="{{ site.baseurl }}/assets/img/lakeFS_integration.png" alt="lakeFS integration into data lake" width="60%" height="60%" />
+* **Copying objects between prefixes to promote new data**
+* **Deleting specific objects to recover from data errors**
+* **Maintaining auxilliary jobs that hydrate a development environment with data**
+
+If you execute any of these actions, adopting lakeFS will speed up development and deployment cycles, reduce the chance of incorrect data making it into production, and make it less painful in the event it does.
+
+Through its versioning engine, lakeFS enables the following built-in operations familiar from git:
+
+* **branch:** a consistent copy of a repository, isolated from other branches and their changes. Initial creation of a branch is a metadata operation that does not duplicate objects.
```

Comment:
Although it's explained in the second sentence, I would be careful using the word _copy_.